blocks/promo/promo.css

```diff
@@ -0,0 +1,96 @@
+.promo {
+  border-radius: var(--udexRadiusL);
+  margin: var(--udexSpacer40) 0;
+  background-color: var(--udexColorGrey1);
+}
+
+.promo.blue {
+  background-color: var(--udexColorBlue1);
+}
+
+.promo>div {
+  display: flex;
+  flex-direction: row;
+  padding: var(--udexSpacer24);
+  gap: var(--udexSpacer24);
+}
+
+@media (width <=640px) {
```

Comment:
This can be `width < 640px` according to this guide
https://sap.frontify.com/document/223141#/layout/grid-breakpoints
Rest looks good to me :)